CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DHeterogeneous.h

```diff
@@ -62,10 +65,12 @@ class TrackingRecHit2DHeterogeneous {
   unique_ptr<TrackingRecHit2DSOAView> m_view;  //!
 
   uint32_t m_nHits;
+  bool m_isUpgrade;
```

Comment:
This appears to be used only in the constructor. Is it necessary to have it as a member variable?

Reply:
Right, this is a typo before `m_nMaxModules` was introduced.